.gitignore

```diff
@@ -8,4 +8,6 @@ yarn.lock
 settings.json
 
 node_modules/
-static/
\ No newline at end of file
+static/
+.venv/
+noah_frontend/dist/
```

Comment:
the folder `dist` has plenty of files that needs to be checked in to version control.

Reply:
okay removing